pkg/wallet/wallet.go

```diff
@@ -123,8 +130,44 @@ func (w *BaseWallet) Close() error {
 }
 
 // CreateDID returns new DID Document
-func (w *BaseWallet) CreateDID() (*did.Doc, error) {
-	return nil, fmt.Errorf("To be implemented")
+func (w *BaseWallet) CreateDID(method string) (*did.Doc, error) {
```

Comment:
`method` needs further refinement: eg. I expect that if I pass something that represents the [`BTCR`](https://w3c-ccg.github.io/didm-btcr/) DID method then I can independently query the bitcoin blockchain and find my DID there.

Reply:
Added TODO